vllm/engine/async_llm_engine.py

```diff
@@ -611,9 +612,12 @@ async def get_model_config(self) -> ModelConfig:
             return self.engine.get_model_config()
 
     @classmethod
-    def from_engine_args(cls,
-                         engine_args: AsyncEngineArgs,
-                         start_engine_loop: bool = True) -> "AsyncLLMEngine":
+    def from_engine_args(
+        cls,
+        engine_args: AsyncEngineArgs,
+        start_engine_loop: bool = True,
+        usage_context: UsageContext = UsageContext.UNKNOWN_CONTEXT
```

Comment:
This should be `ENGINE_CONTEXT` since directly using engine is technically an entrypoint